README.md

```diff
@@ -107,15 +107,16 @@ redis-server
 Sidekiq is the queueing system that ActiveJob interfaces with. Sidekiq should be installed when you ran `bundle install` in the application installation instructions. To start Sidekiq, and make it aware that it is responsible for the mailers queue, execute the following:
 
 ```
-bundle exec sidekiq -q default -q mailers -q exports
+bundle exec sidekiq -q default -q mailers -q exports -q assessments
 ```
 
 ##### Jobs
 
   The following jobs are configured to run continuously:
   * `ConsumeAssessmentsJob`
-      - Should always be running in order to be ready to consume assessments at any time.
-      - Handles consuming assessments from the assessment container into the enrollment container.
+      - Should always be running to be ready to consume assessments at any time
+      - Handles the processing of incomming Patient assessments from the assessment container into the enrollment container.
+      - Utilizes Redis `RPOPLPUSH` pattern.
       - Run with `bin/bundle exec rake reports:receive_and_process_reports`
```

Comment:
should be `bin/bundle exec rake reports:queue_reports` now

Reply:
dd041dc0